### PR TITLE
Add FAQ on missing required assets to debug

### LINKED ISSUE #6408: Missing command to add the missing assets to build and debug your app
There seems to be a missing command to add the missing assets to build and debug the app. Is VS Code limited in this way such that the only way to trigger it to create a default .vscode folder and associated files is to open the folder after creating the app in a terminal? It's just an annoyance to restart VS Code to get the prompt to show, and if an external terminal is required then that defeats some of the utility of having an integrated terminal. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8cc2e9c2-f333-f6a1-064e-790d950b684c
* Version Independent ID: 25a4c9b9-2808-e296-df03-5837000dcbad
* Content: [Get started with C# and Visual Studio Code - C# Guide](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code)
* Content Source: [docs/core/tutorials/with-visual-studio-code.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/with-visual-studio-code.md)
* Product: **dotnet-core**
* GitHub Login: @kendrahavens
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by kendrahavens**
## Summary

Adding instructions for folks who first miss the prompt to add build and debug assets in an FAQ section.

Fixes #6408
 
@mairaw Let me know if there is a better place this info can go. I think this feedback bubbles up often enough that adding a short FAQ section to the doc is worth it. Also in [Omnisharp-vscode repo #1040](https://github.com/OmniSharp/omnisharp-vscode/issues/1040)

### CODE REVIEW

**Comment by mairaw** [comment]
I think that adding the FAQ on that page is fine @kendrahavens!

@gregvanl do you cover this anywhere on the VS docs?

Also because key bindings vary by OS and we don't have that cool feature that VS docs has for different keyboard combinations depending on OS, I think we should give the instruction using the menu **View** > **Command Palette**.

**Comment by gregvanl** [comment]
@maira Not that I am aware. It could also be added to https://code.visualstudio.com/docs/languages/csharp#_common-questions. In general, the VS Code C# is mostly for showing the features (value proposition) of the C# extension. It isn't really a tutorial.

**Comment by mairaw** [comment]
That seems like a good idea @gregvanl. @kendrahavens what do you think?

**Comment by kendrahavens** [comment]
Sounds great! I can add this section there as well. I'll change the keyboard shortcut part to just use the menu.

**Comment by BillWagner** [comment]
@kendrahavens The best answer is to move forward with your plan. (I always just edited the text, not knowing the command exists). So, I merged #6495 based on my (faulty) understanding of the VS Code command palette.

File: docs/core/tutorials/with-visual-studio-code.md [hunk 1]
```diff
@@ -82,6 +82,11 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 > [!TIP] 
 > For more information and troubleshooting tips on .NET Core debugging with OmniSharp in Visual Studio Code, see [Instructions for setting up the .NET Core debugger](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md).
 
+## FAQ
+
+### I'm missing required assets to build and debug C# in VSCode. My debugger says "No Configuration."
```
[the remark this answers is not in the document]
Comment (kendrahavens) [reply docs/core/tutorials/with-visual-studio-code.md]: It is not! Thanks for the catch!

**Comment by kendrahavens** [comment]
@BillWagner Gotcha! I wasn't sure if there was a step I was missing that the copying and pasting took care of! I'll continue forward with the command method.

**Comment by kendrahavens** [comment]
Could I maybe get a review? It's required before merging. @JRAlexander @BillWagner Thanks folks!

File: docs/core/tutorials/with-visual-studio-code.md [hunk 2]
```diff
@@ -82,36 +82,10 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
...
-    "stopAtEntry": false,
-    "internalConsoleOptions": "openOnSessionStart"
-},
-```
-
-Debugging in Visual Studio Code should work after that change.
+## FAQ
+
+### I'm missing required assets to build and debug C# in Visual Studio Code. My debugger says "No Configuration."
+The Visual Studio Code C# extension can generate assets to build and debug for you. If you missed the prompt when you first opened a new C# project, you can still run this command by opening the Command Palette (**View > Command Palette**) and typing ">.NET: Generate Assets for Build and Debug". Selecting this will generate the .vscode, launch.json, and tasks.json configuration files that you need.
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio-code.md]: Nit: I'd rephrase the 2nd sentence as follows: "Visual Studio Code prompts you to generate these assets when you first opened the C# project. If you didn't generate assets then, you can still run this command by ...."

Finally sentence: I recommend changing this to present tense: "Selecting this generates the .vscode..."

File: docs/core/tutorials/with-visual-studio-code.md [hunk 3]
```diff
@@ -82,36 +82,10 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
...
-    "args": [],
-    "cwd": "${workspaceFolder}",
-    "console": "internalConsole",
-    "stopAtEntry": false,
-    "internalConsoleOptions": "openOnSessionStart"
-},
-```
-
-Debugging in Visual Studio Code should work after that change.
+## FAQ
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio-code.md]: Ping @mairaw 

Do these headings match the style guide? If not, what would be the best way to capture this (The SEO value for the full question is great.)

**Review by BillWagner — COMMENTED** [review]
Thanks for adding this @kendrahavens 

I had a couple comments, and I'm requesting that @mairaw take a look at the headings to check against the style guide.

**Review by BillWagner — APPROVED** [review]
And, just looked at @mairaw 's earlier comment on the headings.

I'll approve this, and :shipit:
